### ISSUE TITLE
Pressing Escape from the block toolbar doesn't work for all blocks

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#11774]
When focus is within a block editable area, `Alt + F10` moves focus to the block toolbar.

Once on the toolbar, pressing Escape should move focus back to the previous caret insertion point or selection.

Seems Escape works in some blocks but doesn't work in other blocks.

For example, in the Quote block it doesn't work when editing the quote (no JS errors). It does work when editing the citation. I'd suspect this happens because the Quote block has 2 editable fields.

However, in the List block it doesn't work at all. Haven't tested all the blocks.

**Comment by afercia** [comment]
Also related to pressing Escape from the toolbar: #6165

**Comment by youknowriad** [comment]
I believe this behavior was removed with the navigation mode. So closing this issue.
